partner_contact_configuration/__manifest__.py

```diff
@@ -7,15 +7,15 @@
     'summary': """
         Adds menu configuration access through the 'contacts' module main menu
         """,
-    'version': '10.0.1.0.0',
+    'version': '10.0.1.0.1',
```

Comment:
No need to change this, will be done by ocabot

Reply:
It was not the case on 2018 :-)